source/manual/alerts/email-alert-api-high-queue-latency.html.md

```diff
@@ -14,10 +14,44 @@ review_in: 6 months
 This means the time it takes for a job to be processed is unusually high. This
 may indicate a problem down the line which is preventing workers from being
 processed. It may also imply the threshold is too low if a large number of
-emails have being sent out due to a content change.
+emails are currently being processed.
+
+#### `delivery_transactional`
+
+The `delivery_transactional` queue is used for when we need to send one time
```

Comment:
This is a bit redundant, since we have [this doc](https://github.com/alphagov/email-alert-api/blob/master/docs/queues.md). Perhaps we could move this text into that?

Reply:
Sure, I think the explanation is fine in the queues doc so have left it as is.